addons/mail_plugin/controllers/mail_plugin.py

```diff
@@ -165,6 +165,10 @@ def log_mail_content(self, model, res_id, message):
             raise Forbidden()
         request.env[model].browse(res_id).message_post(body=message)
 
+    @http.route('/mail_plugin/translations', type="json", auth="outlook", cors="*")
```

Comment:
I would use same naming for route and method, it helps grepping (probably mail_plugin/get_translations then).

Reply:
Changed to get translations.

Reply:
@std-odoo this would require changes from your side :)